core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -694,6 +695,14 @@ public ServerBuilder meterRegistry(MeterRegistry meterRegistry) {
         return this;
     }
 
+    /**
+     * Sets a global naming convention for the name of services.
+     */
+    public ServerBuilder serviceNaming(Function<String, String> serviceNaming) {
```

Comment:
How about introducing `ServiceNaming` interface in order to provide default implementations through it.

Reply:
I had a plan to improve with `ServiceNamingRule` at next phase. Thus, I used identity function as a default. Do you think that it would be better to provide an interface in this step?

Reply:
`serviceNaming(ServiceNamingRule)` might be good for self-documentation which will explain how to use this API.
If we apply the interface later, it could be a breaking change.

Reply:
> breaking change

I missed that point. Agreed for sure! Let me update.